### DISCUSSION TITLE
LURO request - add available Balance

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#19] [Bugs and Reports]
Add number of the available balance of tokens. Because its hidden. 

<img width="300" alt="Snímek obrazovky 2024-09-30 v 10 13 09" src="https://github.com/user-attachments/assets/142a4f9e-c4f6-4f93-9722-40f3ab336790">

**Comment by bf-gabagool** [comment]
Hey @JackHipeek 
 We're sorting out old discussions atm and got to this request
 Can you clarify what you mean?
 As far as we understand, you can see your available balance of tokens in the left bar of the page. Did you mean changing 100% with the balance?
 
 Thank you for your message!

**Comment by bf-monster** [comment]
Closing as not needed
